packages/web3/test/shared_fixtures/build/Basic.json

```diff
@@ -0,0 +1,253 @@
+{
```

Comment:
Would be preferred to create a symlink to the `build` folder or particular file.

Reply:
I think, parent folder `shared_fixtures` should be a symlink

Reply:
we have `system_tests_utils.ts` in `shared_fixtures`, which is not in the original folder (@avkos )